### DISCUSSION TITLE
Posted the Pull Requests

### DISCUSSION THREAD

**Discussion by matty10s** [matty10s/group3-project#7] [General]
Hey team, Nice work on the Pull requests.  I have posted the Pull Requests

**Comment by matty10s** [comment]
I will work on Updating the content pages tomorrow.

**Comment by tay-baum** [comment]
Awesome, thanks Matt! I think we can get rid of the plot.html file, Shannon's episodes.html will replace it in the updated nav links.
If we can work on getting our content in on pages this week, I'd like to get the mobile responsive stuff set up on Thursday or Friday so we have some time Saturday to put final touches on.

**Comment by shannonkuhn92** [comment]
Hey y'all. I got all the home page content in tonight and made a few updates to the stylesheet til things looked good. The pull request is in. I'll be doing the trivia page tomorrow!

**Reply by shannonkuhn92** [reply]
Trivia page is also in now. Couldn't figure out how I wanted to do the style for some of it, though. Will mess with it tomorrow.
